lib/authz/permissions.go

```diff
@@ -443,7 +462,13 @@ func (a *authorizer) authorizeAdminAction(ctx context.Context, authContext *Cont
 		return trace.Wrap(err)
 	}
 
-	if _, _, err := a.accessPoint.ValidateMFAAuthResponse(ctx, mfaResp, authContext.User.GetName(), false); err != nil {
+	if a.mfaAuthentictor == nil {
+		return trace.AccessDenied("failed to validate MFA auth response, authorizer missing mfaAuthenticator field")
```

Comment:
Shouldn't this be an Internal-like error? Eg:

```suggestion
		return trace.Wrap(errors.New("failed to validate MFA auth response, authorizer missing mfaAuthenticator field"))
```